scene/gui/margin_container.cpp

```diff
@@ -73,13 +80,20 @@ void MarginContainer::_notification(int p_what) {
 				continue;
 			if (c->is_set_as_toplevel())
 				continue;
+
 			int w;
 			if (c->get_h_size_flags() & Control::SIZE_EXPAND)
```

Comment:
Requiring SIZE_EXPAND to expand the child actually breaks some stuff in the editor... Should I add the EXPAND size flag where needed in the editor or revert this lines?